src/main/java/Deadline.java

```diff
@@ -0,0 +1,33 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class Deadline extends Task {
+    LocalDateTime by;
```

Comment:
Variables should be non-public.
```suggestion
    private LocalDateTime by;
```